components/core/src/clp/ffi/ir_stream/UnstructuredIrDeserializerImpl.cpp

```
@@ -0,0 +1,300 @@
+        IRErrorCode_Success != err)
+    {
+        return ir_error_code_to_errc(err);
+    }
+
+    m_previous_timestamp += timestamp_delta;
+
+    auto encoded_text_ast_result{build_encoded_text_ast<ir::four_byte_encoded_variable_t>(
+            logtype,
+            dict_vars,
+            std::move(encoded_vars)
+    )};
```

Cppcheck (2.20.0)

[style] The function 'serialize_clp_string' is never used.

(unusedFunction)